### PR TITLE
Complete the `README.md` file

### LINKED ISSUE #3: Update the README.md

### PR DESCRIPTION
**Pull request by Hywan**
Fix #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +27,26 @@ install](http://hoa-project.net/Source.html).
...
+line pattern:
+
+```sh
+$ hoa <library-name>:<command-name> <options> <inputs>
+```
+
+Running `hoa` with no argument will list all the available commands with a small
+description. Note: If the option `--no-verbose` is present, the list of commands
+will not be formatted. Thus, used in conjunction with Zsh resources present in
+[`Hoa\Devtools`](http://central.hoa-project.net/Resource/Library/Devtools/Resource/Zsh), you
```
Comment (jubianchi) [inline README.md:40]: Why not moving these resources to the Cli library ? 

Requiring another lib just to get the completion for the current lib seems weird or is it just me ?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ Moreover, Hoa aims at being a bridge between industrial and research worlds.
 
 # Hoa\Cli ![state](http://central.hoa-project.net/State/Cli)
 
-This meta-library provides the `hoa` command line.
+This meta-library provides the `hoa` command line. This is a shell tool to
+access to libraries' commands.
```
Comment (jubianchi) [inline README.md:9]: "to access libraries' commands" ?

or

"to gain access to libraries' commands" ?

**Comment by jubianchi** [comment]
:+1: :+1: :+1: :+1: :+1: :+1:

File: README.md (hunk 1 above)
Comment (Hywan) [inline README.md:40]: This is really related to developer tools, so it is part of another package. `Hoa\Devtools` is really a place for that kind of files.

**Pull request merged**